src/react-components/src/__tests__/components/EditionCard-test.tsx

```diff
@@ -2,10 +2,10 @@ import { expect } from "chai";
 import * as Enzyme from "enzyme";
 import * as React from "react";
 
-import EditionCard from "../../components/02-molecules/Cards/EditionCard";
-import BasicLink from "../../components/01-atoms/Links/BasicLink";
+import EditionCard from "../../components/EditionCard/EditionCard";
+import BasicLink from "../../components/BasicLink/BasicLink";
```

Comment:
Are there situations where we'd want to have a hierarchy like `Link` to group different components?

Reply:
Yeah, seems totally reasonable to want that eventually. For right now though I pulled all of these apart so that we can delete and combine components that should be deleted/combined.